forms/models/elements/subform.js

```diff
@@ -195,9 +195,6 @@ define(function (require) {
                 return memo;
               }
 
-              // let the element know who its parent is
-              element.parentElement = self;
```

Comment:
Huh. So we were setting this on both the SubForm and the SubFormElement. Weird.

Is it worth double-checking this with @samblinkmobile to see if any of the code he has written depends on the `SubFormElement#parentElement` property? We don't depend on it internally, it seems, as the tests pass.